### ISSUE TITLE
Command execution not working correctly

### ISSUE THREAD

**Issue by Vehmloewff** [Vehmloewff/custom-format#4]
See #3

```
[error] Received an error while running the following format command on file.py:
> sed 's/bla/bli/'
Error:
> Command failed with exit code 1: sed 's/bla/bli/'
> sed: -e expression #1, char 1: unknown command: `''
```

```
[error] Received an error while running the following format command on file.py:
> head $FILE
Error:
> Command failed with EPIPE: head file.py
> write EPIPE
```

**Comment by ikoruk** [comment]
I get the same thing with a command which works from the command line:
```
> Command failed with EPIPE: /usr/local/bin/qmlformat [...]/test.qml
> write EPIPE
```

**Comment by Vehmloewff** [comment]
Fixed in 1.0.3
